src/main/java/org/wso2/carbon/connector/googlefirebase/FirebaseConfig.java

```diff
@@ -80,19 +80,27 @@ public void connect(MessageContext messageContext) {
                     log.debug("Started to initialize the firebase admin sdk with the provided firebase service " +
                             "account's credentials");
                 }
+                log.info("Started to initialize the firebase admin sdk with the provided firebase service " +
+                        "account's credentials");
                 FirebaseOptions options = FirebaseOptions.builder()
                         .setDatabaseUrl(FirebaseConstants.HTTPS + credentials.get(FirebaseConstants.API_PROJECT_ID)
                                 + FirebaseConstants.FIREBASE_IO)
                         .setCredentials(GoogleCredentials.fromStream(new ByteArrayInputStream(
-                                credentials.toString().getBytes())))
+                                credentials.toString().replaceAll("\\\\n", "\\n").getBytes())))
                         .build();
+                if(!FirebaseApp.getApps().isEmpty()) {
```

Comment:
Shall we format this , there need to be a space next to of .

Reply:
Fixed.